src/pocketmine/event/block/SignChangeEvent.php

```diff
@@ -79,7 +79,7 @@ public function getLine($index){
 	 * @param string[] $lines
 	 */
 	public function setLines(array $lines){
-		if(count($lines) > 4){
+		if(count($lines) === 4){
```

Comment:
I disagree. I think it's better to push empty values in base of array size. 
For example, if I set an array of 2 lines, the method automatically detect how many empty string need to be pushed inside the lines.